gui/default/syncthing/core/syncthingController.js

```diff
@@ -1586,11 +1607,17 @@ angular.module('syncthing.core')
 
         $scope.saveIgnores = function () {
             if (!$scope.editingExisting) {
+                $scope.newIgnores.edited = true;
+                $scope.newIgnores.ignores = $('#editIgnores textarea').val().split('\n')
                 return;
```

Comment:
Just for my education: What is the advantage of adding an addtional function and button for save?
My reasoning was that `saveIgnores` already handles editing an existing function. I simply extended it to not just stop, but actually process the input in case of editing an existing folder.